distribution/docs/src/main/resources/content/_managing/_installing/installing-from-admin-console.adoc

```diff
@@ -5,7 +5,7 @@
 :project: ${branding}
 :order: 03
 
-===== Completing Installation from the ${admin-console}
+==== Completing Installation from the ${admin-console}
```

Comment:
![image](https://user-images.githubusercontent.com/8041246/52310672-726fb200-2961-11e9-8e50-b3d0774954d6.png)
✏️Because `Firewall Port Configuration` and `Internet Explorer 11 Enhanced Security Configuration` should still be completed regardless of whether installation is completed form the Admin or Command Console, I suggest updating the section intro here
![image](https://user-images.githubusercontent.com/8041246/52310712-94693480-2961-11e9-9c25-12da991010a8.png)

Reply:
👍